CMakeLists_files.cmake

```diff
@@ -138,6 +139,16 @@ list (APPEND TEST_DATA_FILES
   tests/rhs3rep.txt
   tests/options_flexiblesolver.json
   tests/options_flexiblesolver_simple.json
+  tests/GLIFT1.DATA
```

Comment:
Just a heads up about a utility. If you want to have everything in one file you can use `opmpack` which is built as part of opm-common:
```
opmpack GLIFT1.DATA > GLIFT1_ONEFILE.DATA
```